src/app/game-state/inventory.service.ts

```diff
@@ -809,10 +809,16 @@ export class InventoryService {
     this.autoSellItems.splice(index, 1);
   }
 
-  useItemStack(itemStack: ItemStack): void {
-    this.useItem(itemStack.item);
+  useItemStack(itemStack: ItemStack, count?:number): void {
+    if(!count){
+      count = 1;
+    }
```

Comment:
```suggestion
  useItemStack(itemStack: ItemStack, count = 1): void {
```

Reply:
I'll probably end up using defaults on the functions and then ensuring it's at least 1 after to avoid the 0 and negative mess.